### ISSUE TITLE
given, and apply....

### ISSUE THREAD

**Issue by gregglind** [mmonteleone/pavlov#4]
```
    given([2,2,4], [5,2,7], [6,-4,2]).
        it("can generate row data tests", function(a, b, c) {
            assert(c).equals(a + b);
        });
```

All good so far!  but when we have a list of lists.... this doesn't seem to work...

```
    given([[2,2,4], [5,2,7], [6,-4,2]]).
        it("can generate row data tests", function(a, b, c) {
            assert(c).equals(a + b);
        });
```

Nor this:

```
    (given.apply(null, [[2,2,4], [5,2,7], [6,-4,2]])).
        it("can generate row data tests", function(a, b, c) {
            assert(c).equals(a + b);
        });
```

Am I doing something wrong here?

**Comment by mmonteleone** [comment]
allows for given() to be passed an array of arrays (closed by 00f12eed1dd742ee63b55cf7d175dc29f947c806)

**Comment by mmonteleone** [comment]
Thanks for identifying this issue.  I've checked in a fix which allows for given() to be passed an array.  Note that your third example using apply will still not work currently.
